jdaviz/configs/specviz/specviz.yaml

```diff
@@ -21,7 +21,6 @@ tray:
   - g-plot-options
   - g-subset-tools
   - g-markers
-  - spectral-extraction
```

Comment:
this was just missed in #3473 when moving from absorbing specviz2d into specviz to the top-level deconfigged app.